### ISSUE TITLE
Trajectories addon: change performance settings programatically

### ISSUE THREAD

**Issue by olegst90** [KSP-KOS/KOS#2953]
Is it possible to add the capability to adjust Trajectories performance settings (cache usage, integration step, max frames/patch) via KOS interface?

**Comment by nuggreat** [comment]
No, because as far as I can tell trajectories does not expose those settings in there API.

**Comment by olegst90** [comment]
> No, because as far as I can tell trajectories does not expose those settings in there API.
Is it this class?
https://github.com/neuoy/KSPTrajectories/blob/master/src/Plugin/API.cs
They seem to expose UpdateTrajectory, can it be added to the inerface?

My problem is, during the descent cached trajectory is  imprecise and I need to use the plugin without cache to get a precise automatic landing. However, it has performance penalty and I wish I could trigger the recalculation programatically only when I need it, be it through disabling cache option or calling update method explicitly.

**Comment by nuggreat** [comment]
Seeing as that was in the API several months before the last update to the trajectories addon was made and said update to the addon was done by one of the trajectories devs I have to assume they had a reason for not including it in the kOS calls.
